recipe/meta.yaml

```diff
@@ -7,10 +7,10 @@ package:
 
 source:
   url: https://github.com/oceanparcels/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 30d3d6f8a0ed65f7999e5454d7067bff79822fc877a3350492306e48076f0de0
+  sha256: 34091ae664dba671b81a9ea1638931c20f04599cfa51ea355690a6552fa08617
```

Comment:
This is like changing a version in place. Ideally one could do a post release but, IMO, dependency changes are drastic enough to warrant a new version. It is not a big deal for conda-forge but now the package here and on PyPI diverged and there are folks using the latter.

Reply:
Thanks for this point. I don't understand why I had to change the sha256. What changed about the release files then? It is still the same v3.0.2 release? Or am I missing something about how releases on GitHub work?